src/locale/properties/shared/shared.en.properties

```diff
@@ -77,3 +77,9 @@ shared.sortResearch=Sort Research Resources
 shared.addPeerReviews=Add Peer Review
 shared.sortPeerReviews=Sort Peer Reviews
 share.loadMore=Load more
+share.sortTitle=Title
```

Comment:
Translations for the other languages are missing, we have the translations for these words in ORCID-Source